playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -37,43 +55,211 @@ def oom_killer_enricher(event: NodeEvent):
         )
         return
 
-    results: PodList = Pod.listPodForAllNamespaces(
-        field_selector=f"spec.nodeName={node.metadata.name}"
-    ).obj
-
-    oom_kills: List[List] = []
-    headers = ["time", "pod", "container", "image"]
-    for pod in results.items:
-        container_id_to_oom_killed_state = {}
-
-        oom_statuses_from_state = filter(is_state_in_oom_status, pod.status.containerStatuses)
-        for status in oom_statuses_from_state:
-            container_id_to_oom_killed_state[status.state.terminated.containerID] = (status, status.state)
-
-        oom_statuses_from_last_state = filter(is_last_state_in_oom_status, pod.status.containerStatuses)
-        for status in oom_statuses_from_last_state:
-            container_id_to_oom_killed_state[status.lastState.terminated.containerID] = (status, status.lastState)
-
-        for v in container_id_to_oom_killed_state.values():
-            status = v[0]
-            state = v[1]
-
-            dt = parse_kubernetes_datetime_to_ms(state.terminated.finishedAt)
-            oom_kills.append([dt, pod.metadata.name, status.name, status.image])
+    if not isinstance(event, PrometheusKubernetesAlert):
```

Comment:
If we change the event to PrometheusKubernetesAlert, we remove this

Reply:
Removed